helm/chart/router/templates/deployment.yaml

```diff
@@ -85,12 +85,12 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: {{ (index .Values.router.configuration "health-check").path | default "/health" }}
```

Comment:
OMG. I can't believe this is being deleted. @o0Ignition0o ... :(

Reply:
At least the prometheus one survived to tell the tale.

Reply:
no highly esteemed deed is commemorated in this yaml file 🙈

Reply:
This was actually impressive and it didn't feel great removing it.  I'm assuming this was hard to figure out! 😭 

I think we can **and should** revert this PR in the future after we put some more work into ports and paths and readiness/liveliness probes!  Let's not forget about this!